var/spack/repos/builtin/packages/easybuild/package.py

```diff
@@ -15,6 +15,7 @@ class Easybuild(PythonPackage):
     pypi = "easybuild/easybuild-4.0.0.tar.gz"
     maintainers("boegel")
 
+    version("4.7.0", sha256="e68c280e3a508965aefbdf234872919f777f739cf4787cb11bb97a56e0cf74a6")
     version("4.0.0", sha256="21bcc1048525ad6219667cc97a7421b5388068c670cabba356712e474896de40")
 
     depends_on("python@2.6:2.8,3.5:", when="@4:", type=("build", "run"))
```

Comment:
Support for Python 2.6 was removed in EasyBuild v4.4.0 (see [release notes](https://docs.easybuild.io/release-notes/#release_notes_eb440)), so this needs an update

Reply:
Could just delete this line entirely, Spack only supports installing Python 3.7+ so there's no reason to restrict it

Reply:
I think I'm going to keep,
```python
depends_on("python@3.5:", type=("build", "run"))
```

Just in case someone is using an external `python@:3.5`. @adamjstewart are there any objections to doing so?

Reply:
No objections, hopefully no one is doing that though